From cc23dd16fedc274e9ae6d08d9002cdc46f4f33fc Mon Sep 17 00:00:00 2001
From: Venu Vadapalli <vvadapalli@netlogicmicro.com>
Date: Fri, 15 Oct 2010 18:06:58 -0700
Subject: [PATCH 153/565] Turn off msgrng configuration

Turn off msgrng configuration for XLP.

Based on Broadcom SDK 2.3.

Signed-off-by: Venu Vadapalli <vvadapalli@netlogicmicro.com>
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/netlogic/common/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/mips/netlogic/common/Makefile b/arch/mips/netlogic/common/Makefile
index 6b8b482..8da2f90 100644
--- a/arch/mips/netlogic/common/Makefile
+++ b/arch/mips/netlogic/common/Makefile
@@ -1,12 +1,12 @@
 EXTRA_CFLAGS := $(CFLAGS) -DNLM_HAL_LINUX_KERNEL -Iarch/mips/include/asm/netlogic/hal
-obj-y :=   platform.o cpu_proc.o memory.o
+obj-y :=   cpu_proc.o memory.o bootinfo.o
 
 obj-y 					+= msgring.o
 obj-y 					+= msgring_xls.o
 obj-y 					+= msgring_shared.o
-obj-$(CONFIG_NLM_XLP)			+= nlm_hal_fmn_config.o
 
-obj-$(CONFIG_NLM_XLP)			+= nlm_hal.o
+obj-$(CONFIG_NLM_XLP)			+= nlm_hal_fmn_config.o
+obj-$(CONFIG_NLM_XLP)			+= nlm_hal.o nlm_hal_nae.o
 
 obj-$(CONFIG_RAPIDIO)             	+= srio.o
 obj-$(CONFIG_NLMCOMMON_IP_OVER_PCI) 	+= dma.o
@@ -16,6 +16,6 @@ EXTRA_AFLAGS := $(CFLAGS)
 
 clean-files += msgring.o msgring_xls.o msgring_shared.o nlm_hal_fmn_config.o
 clean-files += nlm_hal.o
-clean-files += srio.o dma.o smp.o platform.o cpu_proc.o
+clean-files += srio.o dma.o smp.o cpu_proc.o
 clean-files += msgring.c msgring_xls.c msgring_shared.c nlm_hal.c
 
-- 
1.8.4.93.g57e4c17

